src/questions.js

```diff
@@ -0,0 +1,143 @@
+const { validateInput } = require("./utils");
+
+const managerQuestions = [
+  {
+    type: "input",
+    name: "teamName",
+    message: "Enter your team's name:",
+    validate: validateInput,
+  },
+  {
+    type: "input",
+    name: "fileName",
+    message: "Enter new file name without an extension:",
+    validate: (fileName) => {
+      return (
+        /(^[\w\-]+)$/.test(fileName) ||
+        "Please enter a valid file name without an extension"
+      );
+    },
+  },
+  {
+    type: "input",
+    name: "name",
+    message: "Manager name:",
+    validate: validateInput,
+  },
+  {
+    type: "input",
+    name: "id",
+    message: "Manager ID number:",
+    validate: (id) => {
+      return /^\d+$/.test(id) || "Please enter only numbers.";
+    },
+  },
+  {
+    type: "input",
+    name: "email",
+    message: "Manager email address:",
+    validate: (email) => {
+      return (
+        /^([a-z0-9_\.-]+)@([\da-z\.-]+)\.([a-z\.]{2,6})$/.test(email) ||
```

Comment:
I'd allow capital letters here as emails are not case sensitive, this is optional though.